src/api/queries/blockchainQueries.js

```diff
@@ -1,165 +1,149 @@
 ApiClient_.prototype.fetchActiveDeposits = function (slug, from, to) {
-  var query = {
-    'query': '{\
-       dailyActiveDeposits(slug: "' + slug + '",\
-                           from: "' + toUTC_(from) + '",\
-                           to: "' + toUTC_(to) + '",\
-                           interval: "1d") {\
-         activeDeposits\
-         datetime\
-       }\
-     }'
+  const query = {
+    'query': `{
+       dailyActiveDeposits(${commonParams(slug, from, to)}) {
```

Comment:
getMetric(metric: "active_deposits")

Reply:
hm, now that I look there are at least 7 queries that should be replaced with getMetric, let's do that in another PR

Reply:
Made a card for it